\documentclass[fleqn,10pt]{wlscirep}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{amsmath,amssymb}
\DeclareMathOperator*{\SIGMA}{SIGMA}
\title{Opening Report of CRISPR/Cas9-based Screening Data Analysis}

\author[1]{Y.Q. Yang}

%\keywords{Keyword1, Keyword2, Keyword3}

\begin{abstract}
In the opening report, I did a brief acadamic research on sgRNA specific pooled library screening, and part of current algorithms including MAGeCK and BAGEL.  Due to the difficulty of understanding these algorithm, I have problem finishing own-data analysis, which should be done in the work that follows. 
\end{abstract}
\begin{document}

\flushbottom
\maketitle
% * <john.hammersley@gmail.com> 2015-02-09T12:07:31.197Z:
%
%  Click the title above to edit the author information and abstract
%
\thispagestyle{empty}

%\noindent Please note: Abbreviations should be introduced at the first mention in the main text â€“ no abbreviations lists. Suggested structure of main text (not enforced) is provided below.

\section*{Background}

In the first week of starting this project, I think it necessary for understanding what CRISPR/Cas9-based genetic screening is, and what should be considered in analyzing the screening data.  The idea of CRISPR/Cas9-based genetic screening was to use a pool of sgRNA-expressing lentivirus to generate a library of knockout cells that could be screened under both positive and negative selection. \cite{wang2014genetic} In addition to transfection, Pooled screening can avoid experimental errors caused by high expression of sgRNA.
However, the data generated by these screens pose several challenges to computational analysis. First, variance and statistical significance of comparisons between sample and control should be calculated under a extremely small sample size.  Second, different sgRNAs target the same gene might have different specificities and knockout efficiencies, which should be taken into account in the algorithm.  Third, the difference between read count distributions are significant, which calls for a robust normalization. \cite{li2014mageck}

\section*{Results}

Detailed information about MAGeCK and BAGEL algorithms, which is concluded partially on my own opinion, is shown below.

\subsection*{Principles of MAGeCK Algorithm}

\subsubsection*{Overview of MAGeCK Algorithm}

A schematic of the MAGeCK Algorithm is presented as follow\cite{li2014mageck}.
\begin{enumerate}
    \item Read counts from different samples are median-normalized, which enables each sample to have the same median value.
    \item A mean-variance model is established by following the \textbf{empirical} equation:
        \begin{equation}
           \hat{\sigma}^2=\hat{\mu}+k\hat{\mu}^b
        \end{equation}
        or
        \begin{equation}
            \log{(\hat{\sigma}^2-\hat{\mu})}=\log{k}+b\log{\hat{\mu}}\textrm{,}\; k\geqslant{0} \textrm{,}\;b\geqslant{0}
         \end{equation}
        In the above formula, $\hat{\sigma}$ represents sample varience, and $\hat{\mu}$ represents sample means.  This approach aims to estimate the varience of the read counts within one condition in a small number of replicates.
    \item The sample mean and varience obtained in the last step can be used as parameters of the negative binomial(NB)distribution, which is applied to rank sgRNA based on P-values calculated from the NB model.  This model can be used in both positive and negative selection of each sgRNA.
    \item Essential genes, which is targete by relatively huger amount of sgRNAs rank near the top of the sgRNA list generated in the last step, are identified using a modified robust ranking aggregation($\alpha$-RRA) algorithm.
    \item Test the enrichment of pathways also 
    \end{enumerate}

\subsubsection*{Algorithm Characteristics}
\textbf{Negative Binomial Algorithm}

This algorithm particularly uses a negative binomial(NB) model to test whether each sgRNA abboudance is significantly different between treatments and control based on the P-value calculated from the model\cite{li2014mageck}. 
For a set of read counts of sgRNA $i$ with replicates in two condition A and B(for example, in CRISPR/Cas9-treated samples and in control samples), the P-value is calculated as follow\cite{li2014mageck}.:
\begin{equation}
    p=\left\{
\begin{aligned}
\sum_{x>\mu_{iB}}NB(x|\mu_{iA}\textrm{,}\;\sigma^2_{iA}) \textrm{,}\;\mu_{iB}>\mu_{iA}\\
\sum_{x<\mu_{iB}}NB(x|\mu_{iA}\textrm{,}\;\sigma^2_{iA}) \textrm{,}\;\mu_{iB}<\mu_{iA} 
\end{aligned}
\right.
\end{equation}
Where $NB(x|\mu_{iA}\textrm{,}\;\sigma^2_{iA})$ is the probability mass function(PMF) of a read count x from the NB distribution with mean $\mu_{iA}$ and $\sigma^2_{iA}$. It is applied to test whether the sgRNA is positively selected($\mu_{iB}>\mu_{iA}$) or negatively selected($\mu_{iB}<\mu_{iA}$).

Considering that the sample size of sgRNA-specific pooled screening is relatively small and discrete, commonly used probablity distribution model such as binomial or Poisson, which is derived from large-sample asymptotic theory, may not appropriately model the count viability in RNA-Seq data.\cite{di2011nbp} Current RNA-Seq methods, including FPKM and TPM, which have been introduced in class, typically normalize data by scaling the number of reads in a given lane or library to a common value across all sequenced libraries in the experiment.  However, library size scaling is too simple for many biological conditions, true biological differences in RNA composition between samples will be the main reason for normalization.  Depending on the experimental situation, Negative Binomial may be appropriate for the additional variation observed from biological replicates\cite{robinson2010scaling}.

\textbf{Robust Ranking Aggregation(RRA) Algorithm}\cite{li2014mageck}

Essential genes are ranked separately using this algorithm.  Suppose $M$ sgRNAs are included in the experiment, and $R=(r_1\textrm{,}\;r_2\textrm{,}\;\cdots \textrm{,}\;r_n)$ is the vector of ranks of $n$ sgRNAs targeting a gene $i(n\ll M\textrm{,}\;r_i\ll M)$.  Firstly, the ranks are normalized into percentiles $U=(u_1\textrm{,}\;u_2\textrm{,}\;\cdots \textrm{,}\;u_n)$, where $u_i=r_i/M(i=1\textrm{,}\;2\textrm{,}\;\cdots\textrm{,}\;n)$. The $k$th smallest value among $u_1\textrm{,}\;u_2\textrm{,}\;\cdots \textrm{,}\;u_n$ is fit to a beta distribution with the null hypotheses as uniform distribution between 0 and 1.  The significance score of the gene, the $\rho$ value, is defined as $\rho=min(\rho_1)\textrm{,}\;\rho_2\textrm{,}\;\cdots \textrm{,}\;\rho_n$, where $\rho_k$ is the P-value, which is based on the beta distribution, of the gene $k$.

To avoid false positive situations, the $\rho$ values should be refined by constricting the range of selected sgRNAs\cite{li2014mageck}.  Also a permutation test should be performed and a False Discovery Rate(FDR) should then be calculated.

Due to the incomparability of different gene expression levels caused by quite a number of factors such as differences in measurement platforms and lab protocols, finding a meaningful combination of different data sources is often a non-trivial task.\cite{doi:10.1093/bioinformatics/btr709}  Consequently, it might be better to analyze data from different genes separately and then aggregate the resulting gene lists. 

\subsection*{Principles of BAGEL Algorithm}

\subsubsection*{Overview of BAGEL Algorithm}\cite{hart2016bagel}
The BAGEL Algorithm; however, is different from MAGeCK Algorithm to some extent.  Other than collecting the raw read count of each sgRNAs, changes in the frequency distribution of each gRNA in the population, which is comparing to that at an early control timepoint, are measured as fold change. Essential and non-essential genes are artificially defined so as to train  reference sets of fold change.  These sets will be used to evaluate the likelihood that the ovserved fold changes for gRNA targeting the gene were drawn from either the essential or the non-essential traning distributions.  In the evalutaion process, a Bayesian classifier is applied to calculate the likelihood result.

Since I don't really trust the training set derived from artificially selected essential genes, as I see it, a new dataset should also be generated as long as we would apply this method. Furthermore, I think the selection of reference essential and non-essential gene data are critical for supervised-machine-learning-like method.

\section*{Future Plan}

For discovering MAGeCK Algorithm:
\begin{itemize}
    \item Learn more about Beta distribution;
    \item Learn about the imporvement which has been made in MAGeCK-VISPR;
    \item Find out the realistic difference between MAGeCK and other algorithms in own-data analysis, especially in the application of NB and RRA;
    \item Analysis the difference between MAGeCK and BAGEL by using datasets retrieved from our own research;
    \item ...
\end{itemize}

For further application of Sequence Screening data analysis:
\begin{itemize}
    \item Try to make some specialized optimization of algorithm for pathway analysis;
    \item Establish new reference datasets to test whether BAGEL Algorithm can perform well in our own field;
    \item ...
\end{itemize}

\bibliography{sample}


\end{document}